angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-accordion-item/sprk-masthead-accordion-item.component.ts

```diff
@@ -130,7 +133,8 @@ export class SprkMastheadAccordionItemComponent implements OnInit {
   public animState = 'closed';
 
   /**
-   * @ignore
+   * This applies the correct styles to the item
+   * when it is open or closed.
```

Comment:
I thought it would be beneficial for users to be able to read the methods of the component and what they do. This method is public by default, so if they wanted, they could trigger this method on the class. So I thought having documentation for it would be beneficial.